test/Microsoft.AspNet.Authentication.Test/OpenIdConnect/OpenIdConnectMiddlewareTests.cs

```diff
@@ -82,7 +82,8 @@ public class OpenIdConnectMiddlewareTests
                 options.Configuration = TestUtilities.DefaultOpenIdConnectConfiguration;
             });
             var transaction = await SendAsync(server, DefaultHost + Challenge);
-            transaction.SetCookie.Single().ShouldContain(OpenIdConnectAuthenticationDefaults.CookieNoncePrefix);
+            transaction.SetCookie.First().ShouldContain(OpenIdConnectAuthenticationDefaults.CookieNoncePrefix);
+            transaction.SetCookie.Skip(1).First().ShouldContain(OpenIdConnectAuthenticationDefaults.CookieStatePrefix);
         }
```

Comment:
I thought about it, but I didn't want to start doing it just for this one test. We'll do a clean sweep later.